compiler/damlc/tests/daml-test-files/InterfaceRetro.EXPECTED.md

```diff
@@ -0,0 +1,91 @@
+# <a name="module-interfaceretro-19712"></a>Module InterfaceRetro
```

Comment:
This should (eventually) have an entry under `## Interfaces -> Token` for the retroactively implemented templates, I added a bullet point for extending daml doc in https://github.com/digital-asset/daml/issues/14047